src/components/common/section-templates/single-title-quintuple-description/SingleTitleQuintupleDescription.module.scss

```
@@ -56,6 +56,7 @@ $desc-input-height: 115px;
     text-align: left;
     font-size: clamp(12px, 2.1vw, 34px);
     line-height: clamp(14px, 2.5vw, 42px);
+    word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)